src/decaylanguage/dec/dec.py

```diff
@@ -655,21 +660,37 @@ def list_decay_modes(self, mother, pdg_name=False):
             for mode in self._find_decay_modes(mother)
         ]
 
-    def _decay_mode_details(self, decay_mode):
+    def _decay_mode_details(self, decay_mode, display_photos_keyword=False):
```

Comment:
```suggestion
    def _decay_mode_details(self, decay_mode, display_photos_keyword):
```

Reply:
This is an internal function and I then find it suitable that the caller is explicit about the "configuration".

Reply:
Any reason why you did not simply "commit suggestion" or "add suggestion to batch" (then having all the suggestions committed in one go) instead of having prepared all the changes 1-by-1 on your side, which is far more work ;-). I tend not to commit immediately myself to give the opportunity for users to see the updates and be able to comment back.